packages/shell-api/src/abstract-cursor.ts

```diff
@@ -53,8 +53,8 @@ export abstract class AbstractCursor<CursorType extends ServiceProviderAggregati
   }
 
   @returnsPromise
-  async close(options: Document = {}): Promise<void> {
-    await this._cursor.close(options);
+  async close(): Promise<void> {
```

Comment:
I wasn't sure if this was okay (I'm not sure if this method gets exposed).  If we need to preserve the method signature, I can skip the "no unused vars" lint rule for this line.

Reply:
Mh yeah … I guess this would be an issue only if we ever would decide to reintroduce an argument for this function (with possibly different semantics), but that seems very unlikely? I think this is fine as it is, tbh.